_posts/en/newsletters/2023-08-16-newsletter.md

```diff
@@ -0,0 +1,126 @@
+---
+title: 'Bitcoin Optech Newsletter #264'
+permalink: /en/newsletters/2023/08/16/
+name: 2023-08-16-newsletter
+slug: 2023-08-16-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME:harding
+
+## News
+
+- **Adding expiration metadata to silent payment addresses:** Peter Todd
+  [posted][todd expire] to the Bitcoin-Dev mailing list a recommendation
+  to add a user-chosen expiration date to addresses for [silent
+  payments][topic silent payments].  Unlike regular Bitcoin addresses
+  that result in [output linking][topic output linking] if used to
+  receive multiple payments, addresses for silent payments result in a
+  unique output script every time they are properly used.  This can
+  significantly improve privacy when it's impossible or inconvenient for
+  receivers to provide spenders with a different regular address for
+  each separate payment.
+
+    Peter Todd notes that it would be desirable for all addresses to
+    expire: at some point, most users are going to stop using a wallet.
+    The expected one-time use of regular addresses makes expiry less of
+    a concern, but the expected repeated use of silent payments makes it
+    more important that they include an expiry time.  He suggests the
+    inclusion of either a two-byte expiry time in addresses that would
+    support expiry dates up 180 years from now or a three-byte time that
+    would support expiry dates up to about 45,000 years from now.
+
+    The recommendation received a moderate amount of discussion on the
+    mailing list, with no clear resolution as of this writing.
+
+- **Serverless payjoin:** Dan Gould [posted][gould spj] to the
+  Bitcoin-Dev mailing list a [draft BIP][spj bip] for _serverless
+  [payjoin][topic payjoin]_.  By itself, [payjoin][topic payjoin] as
```

Comment:
Consider a (see Newsletter #236) link to https://bitcoinops.org/en/newsletters/2023/02/01/#serverless-payjoin-proposal